src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Dec2Hex.txt

```diff
@@ -177,16 +183,31 @@ Table({Value:"1"},{Value:"0"},{Value:Error({Kind:ErrorKind.Numeric})},{Value:"2"
 >> Dec2Hex([10,11,"12"], [2,5,10])
 Table({Value:"0A"},{Value:"0000B"},{Value:"000000000C"})
 
+>> Dec2Hex(["10","11","12"])
+Table({Value:"A"},{Value:"B"},{Value:"C"})
+
+>> Dec2Hex(["10","11","12"], [2,5,10])
+Table({Value:"0A"},{Value:"0000B"},{Value:"000000000C"})
+
 >> Dec2Hex([10,255,-16], 5)
 Table({Value:"0000A"},{Value:"000FF"},{Value:"FFFFFFFFF0"})
 
 >> Dec2Hex(10, [2,5,10])
 Table({Value:"0A"},{Value:"0000A"},{Value:"000000000A"})
 
+>> Dec2Hex(255, [Blank(),-1,0,1,2,3,4])
+Table({Value:Error({Kind:ErrorKind.Numeric})},{Value:Error({Kind:ErrorKind.Numeric})},{Value:Error({Kind:ErrorKind.Numeric})},{Value:Error({Kind:ErrorKind.Numeric})},{Value:"FF"},{Value:"0FF"},{Value:"00FF"})
+
 // ******** DIFFERENT INPUT TYPES AS PARAMETERS ********
 
 >> Dec2Hex(Table({a:10},{a:255},{a:-16}))
 Table({Value:"A"},{Value:"FF"},{Value:"FFFFFFFFF0"})
 
 >> Dec2Hex(Table({a:10},{a:11},{a:12}), Table({b:2},{b:5},{b:10}))
 Table({Value:"0A"},{Value:"0000B"},{Value:"000000000C"})
+
+>> Dec2Hex(Table({A: 12}), 3)
+Table({Value:"00C"})
+
+>> Dec2Hex(Table({A: 12, B:13}))
+Errors: Error 8-28: Invalid argument type (Table). Expecting a Number value instead.
```

Comment:
>Invalid argument type (Table). Expecting a Number value instead. [](http://example.com/codeflow?start=20&length=64)

This error message is misleading. The problem is that we require a one-column table. See the tests for Hex2Dec - they have a good error message

Reply:
per offline discussion, we need to keep this as of now we don't have a way to pick the right overload function when the function doesn't specify default argument types like Dec2Hex, Power, Log, etc